### ISSUE TITLE
TO_DO : use textures and colors given with map

### ISSUE THREAD

**Issue by Alicia-bites** [Alicia-bites/cub3d#11]
Right now the char *no, so, we, and ea in struct `settings` are empty --> 

# CODE
```c
printf("so = %s\n", setup.so);
printf("no = %s\n", setup.no);
printf("we = %s\n", setup.we);
printf("ea = %s\n", setup.ea);
```
# OUTPUT
```c
so = (null)
no = (null)
we = (null)
ea = (null)
```

So I can't use them ^^

And also I will need something for the floor color and ceiling color but maybe you did have variables for that in that case I'm sorry and I will kill myself for the shame I bring upon my family.

**Comment by BarrierAntho** [comment]
Have a look to settings structure ;)
```
	int		f_r;
	int		f_g;
	int		f_b;
	int		c_r;
	int		c_g;
	int		c_b;
```
each value is the separate rgb value
f_r, f_g, f_b = floor rgb
c_r, c_g, c_b = ceil rgb

**Comment by BarrierAntho** [comment]
fixed a50dfd354ddebfe1be8b71f4cb34e791c24cb5f5
Test with your own test ;)
